modules/nf-core/taxpasta/standardise/meta.yml

```
@@ -0,0 +1,57 @@
+name: "taxpasta_standardise"
+description: "Standardise the output of a wide range of taxonomic profilers"
+keywords:
+  - "taxonomic profile"
+  - "standardise"
+tools:
+  - "taxpasta":
+      description: "TAXonomic Profile Aggregation and STAndardisation"
+      homepage: "https://taxpasta.readthedocs.io/"
+      documentation: "https://taxpasta.readthedocs.io/"
+      tool_dev_url: "https://github.com/taxprofiler/taxpasta"
+      doi: ""
+      licence: "['Apache-2.0']"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  #
+  - profile:
+      type: file
+      description: profiler output file
+      pattern: "*.{txt,tsv,out}{.gz}"
+  #
+  - output_format:
+      type: val
+      description: The file extension for the output format
+      pattern: "{tsv,csv,arrow,parquet,biom}"
+  #
+  - taxonomy:
+      type: directory
+      description: Directory containing at a minimum nodes.dmp and names.dmp files
+      pattern: "*/"
+
+output:
+  #Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  #
+  - profile:
```

See suggestion above, but at least needs to be
```suggestion
  - profiles:
```

if you stick with the original channel name